Sources/WebService/NetworkResource+Modifiers.swift

```diff
@@ -128,11 +128,12 @@ public extension NetworkResource {
     /// Mocks the response of the resource with the contents of the given filename. Note that if a request is mocked, it'll never hit the network, and will NOT pass the Request Interceptors. It will, however, pass through the Response Intereptors.
     ///
     /// - Parameter withFile: The name (without extension) of the file containing the mocked response. The file must be present in the main bundle (`Bundle.main`)
+    /// - Parameter inBundle: Bundle in which mock file is located. Defaults to main bundle
     /// - Parameter ofType: The extension of the file. Defaults to `.json` if not provided.
     /// - Returns: NetworkResource
-    @objc @discardableResult func mock(withFile: String, ofType: String = "json") -> NetworkResource {
-        guard let path = Bundle.main.path(forResource: withFile, ofType: ofType), let data = try? Data(contentsOf: URL(fileURLWithPath: path)), data.count > 0 else {
-            print("[Swifty] Unable to mock response from file: \(withFile).\(ofType): Make sure the filename and extension are correct, and the file is present in the main bundle of your app. Also make sure the file is not empty.")
+    @objc @discardableResult func mock(withFile: String, inBundle: Bundle = Bundle.main, ofType: String = "json") -> NetworkResource {
```

Comment:
Can make this `func mock(withFile: String, inBundle bundle: Bundle = Bundle.main, ofType: String = "json")`, so that the variable for bundle inside the function is still `bundle`, and not `inBundle`. Overall, I think this is good idea.